app/ide-desktop/lib/dashboard/src/authentication/src/authentication/service.tsx

```diff
@@ -0,0 +1,99 @@
+/** @file Provides an {@link AuthService} which consists of an underyling {@link Cognito} API
+ * wrapper, along with some convenience callbacks to make URL redirects for the authentication flows
+ * work with Electron. */
+import * as cognito from "./cognito";
+import * as authConfig from "./config";
+import * as config from "../config";
+
+
+
+// =================
+// === Constants ===
+// =================
+
+const BASE_AMPLIFY_CONFIG: Partial<authConfig.AmplifyConfig> = {
+    region: authConfig.AWS_REGION,
+    scope: authConfig.OAUTH_SCOPES,
+    responseType: authConfig.OAUTH_RESPONSE_TYPE,
+};
+
+/** Collection of configuration details for Amplify user pools, sorted by deployment environment. */
+const AMPLIFY_CONFIGS = {
+    /** Configuration for @pbuchu's Cognito user pool. */
+    pbuchu: {
+        userPoolId: "eu-west-1_jSF1RbgPK",
+        userPoolWebClientId: "1bnib0jfon3aqc5g3lkia2infr",
+        domain: "pb-enso-domain.auth.eu-west-1.amazoncognito.com",
+        ...BASE_AMPLIFY_CONFIG,
```

Comment:
What is ` @pbuchu's Cognito user pool` ? I don't think this should be merged, as it looks like a local config of Pawels.

Reply:
what about something like `process.env.USER_POOL_ID ?? "eu-west-1_9Kycu2SbD" /* fallback to production user pool */`?

Reply:
Yeah the `process.env` approach doesn't work great unless we're running in `npm run dev`. So it doesn't work for either the Electron distribution or the static cloud files. I think we can keep the existing configs as-is.